### PR TITLE
Fixes DOC-2264

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2264](https://openedx.atlassian.net/browse/DOC-2264)

Updates existing tool descriptions to include accessibility of remote material as a consideration.
### Reviewers
- [X] Subject matter expert: @cptvitamin 
- [x] Doc team review (copyedit): @catong @srpearce 
### Testing
- [X] make html ran without unexpected errors
  ### Post-review
- [x] Squash commits

### CODE REVIEW

**Comment by cptvitamin** [comment]
@lamagnifica thanks for all the fixes and for referencing Best Practices where third party content is involved. :+1:

File: en_us/shared/exercises_tools/google_calendar.rst [hunk 1]
```diff
@@ -28,22 +28,29 @@ You can also use Google Drive files, such as documents, spreadsheets, and
 images, in your course. For more information, see :ref:`Google Drive Files
 Tool`.
 
-.. note:: Google services are not available in some regions and countries. If 
- Google services are not available in a learner's area, the learner might see
- an “image unavailable” message in the place of the Google Drive file or
- calendar. We strongly suggest that you provide alternative resources
- for learners in these areas.
+Before you include a Google calendar in your course, be sure to evaluate the
+following considerations.
```
Comment (srpearce) [inline en_us/shared/exercises_tools/google_calendar.rst]: I'd say something like "keep the following considerations in mind" - the current version sounds a little stilted.

File: en_us/shared/exercises_tools/google_calendar.rst [hunk 2]
```diff
@@ -28,22 +28,29 @@ You can also use Google Drive files, such as documents, spreadsheets, and
...
+
+* Google services are not available in some regions and countries. If Google
+  services are not available in a learner's area, the learner might see an
+  "image unavailable" message in the place of the Google Drive file or
+  calendar. We strongly suggest that you provide alternative resources for
+  learners in these areas.
+
+* Review all supplemental materials to assure that they are accessible before
+  making them available through your course. For more information, see
+  :ref:`Accessibility Best Practices for Course Content Development`.
```
Comment (srpearce) [inline en_us/shared/exercises_tools/google_calendar.rst]: These two bullets aren't parallel (first is declarative, second is imperative) - I'd change one of them.

File: en_us/shared/exercises_tools/google_docs.rst [hunk 3]
```diff
@@ -27,22 +27,31 @@ following topics.
 You can also use :ref:`Google calendars<Google Calendar Tool>` in courseware.
 For more information, see :ref:`Google Calendar Tool`.
 
-.. note:: Google services are not available in some regions and countries. If 
- Google services are not available in a learner's area, the learner might see
- an “image unavailable” message in the place of the Google Drive file or
- calendar. We strongly suggest that you provide alternative resources
- for learners in these areas.
```
Comment (srpearce) [inline en_us/shared/exercises_tools/google_docs.rst:34]: Same comments as above for the following paragraphs.

File: en_us/shared/exercises_tools/recommenderXBlock.rst [hunk 4]
```diff
@@ -18,14 +18,14 @@ OpenStax, as well as in the course itself. The Recommender can help fill
...
-
-   * Add new resources.
-   * Edit existing resources and work jointly to improve the quality of
-     resources (e.g., give an informative resource summary).
-   * Manage quality by voting for high quality resources or flagging
-     resources as spam or abuse.
+* Add new resources.
+* Edit existing resources and work jointly to improve the quality of resources
+  (for example, give an informative resource summary).
+* Manage quality by voting for high quality resources or flagging resources as
```
Comment (srpearce) [inline en_us/shared/exercises_tools/recommenderXBlock.rst:25]: high-quality

**Comment by srpearce** [comment]
Just a few comments - otherwise :+1:

File: en_us/shared/exercises_tools/recommenderXBlock.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/recommenderXBlock.rst:25]: FYI, this is not in our wordlist.rst as an exception to the minimal hyphenation rule...

File: en_us/shared/exercises_tools/google_calendar.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/google_calendar.rst]: agreed!

File: en_us/shared/exercises_tools/google_calendar.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/google_calendar.rst]: @srpearce nothing is coming to mind, suggestions?

Comment (srpearce) [inline en_us/shared/exercises_tools/google_calendar.rst]: Maybe change the second one:

All supplemental materials must be accessible (SP: if this is true - if not, just delete this sentence and start with the next one). EdX strongly encourages you to review your supplemental materials for accessibility before you make them available in your course. For more information...

**Pull request merged**